src/main/java/interface_adaptors/ExportController.java

```diff
@@ -0,0 +1,18 @@
+package interface_adaptors;
+
+import use_cases.ExportDS;
+import use_cases.ExportInputBoundary;
+
+public class ExportController {
+    ExportInputBoundary exportUseCase;
+
+    public ExportController(ExportInputBoundary useCase){
+        this.exportUseCase = useCase;
+    }
+    public String export(){
+        ExportDS data = exportUseCase.extractDataStorage();
+        gatewayWriter writer = new gatewayWriter("src/main/java/database/Database.csv");
+        writer.rewriteFile(data.getDatabase());
+        return "src/main/java/database/Database.csv";
```

Comment:
It returns a string here so I guess it stores the csv file in this path. Does that mean we must have this file path in our laptop?